modules/pugixml/1.14/patches/add_build_file.patch

```diff
@@ -0,0 +1,17 @@
+--- /dev/null
++++ b/BUILD.bazel
```

Comment:
since patch_strip is 0 in your source.json file, this line needs to be just `BUILD.bazel`, instead of `b/BUILD.bazel`

Reply:
```suggestion
+++ BUILD.bazel
```